app/controllers/users_controller.rb

```diff
@@ -27,6 +27,8 @@ def update
   def destroy
     authorize @user
     @user.destroy
+    @identity = Identity.find(params[:id]) if params[:id]
+    @identity.destroy
```

Comment:
Don't use the `UsersController` to destroy an identity. I don't think we need to worry about deleting an identity when deleting a user, but if we do, we can do that from the `User` model.